packages/app/src/templates/selectors/templates-list.selector.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { createSelector } from "reselect"
+import { ReduxRootState } from "App/__deprecated__/renderer/store"
+import { Template } from "App/templates/dto"
+
+const templatesSelector = ({ templates }: ReduxRootState) => templates.data
+
+const sortTemplates = (templates: Template[]): Template[] => {
+  return [...templates].sort((a, b) => {
+    const orderA = a.order
+    const orderB = b.order
+    return orderA - orderB
+  })
+}
```

Comment:
I like to give the variables the meaning name, but seems to me that in that case it is a little overkill.

Not a big thing but maybe it should be changed to one of those (I like more the first one)?
```
const sortTemplates = (templates: Template[]): Template[] => {
  return [...templates].sort((a, b) => {
    return a.order - b.order
  })
}
const sortTemplates2 = (templates: Template[]): Template[] => {
  return [...templates].sort(({ order: orderA }, { order: orderB }) => {
    return orderA - orderB
  })
}
```

Reply:
Changed to first solution in [43e06d1](https://github.com/mudita/mudita-center/pull/804/commits/43e06d134aad31f5e351a09965e5574fbf64604f)